<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="TicketManager">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="route">
       <attribute name="title">
        <string>Маршрут</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="3" column="0">
         <widget class="QComboBox" name="cb_routes">
          <item>
           <property name="text">
            <string>Москва-Новосибирск</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Красноярск-Иркутск</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Казань-Самара</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Кострома-Ярославль</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Тюмень-Екатеринбург</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Якутск-Владивосток</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Направление</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QRadioButton" name="rb_there">
             <property name="text">
              <string>Туда</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rb_roundTrip">
             <property name="text">
              <string>Туда-обратно</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string> Выбор маршрута</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Добавить маршрут</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Данные пассажира</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Данные пассажира</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="lb_firstName">
             <property name="text">
              <string>Имя</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="le_FirstName"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lb_lastName">
             <property name="text">
              <string>Фамилия</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="le_LastName"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lb_burthDate">
             <property name="text">
              <string>Дата рождения</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="de_birthDate"/>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="lb_addPassenger">
             <property name="text">
              <string>Добавить пассажира</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QProgressBar" name="PBPassengers">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Результат</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <widget class="QPushButton" name="pb_enableLog">
          <property name="text">
           <string>Добавлять вывод пассажиров на экран</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Список пассажиров</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QTextEdit" name="te_passengers"/>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pb_clear">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Маршрут</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="le_route"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
